tests/test_wog.py

```diff
@@ -96,6 +96,17 @@ def test_hallazeal_the_ascended():
 	assert hallazeal.dead
 
 
+def test_mire_keeper():
+	game = prepare_game()
+	game.player1.give("OG_202").play(choose="OG_202a")
+	assert len(game.player1.field) == 2
+	game.end_turn()
+	game.end_turn()
```

Comment:
Stylistic fix, whenever you want to do a full turn cycle, do `game.end_turn(); game.end_turn()`. Also make sure to follow it up with a blank line so we can quickly see the turns.